minidump-processor/src/processor.rs

```diff
@@ -164,6 +164,13 @@ where
         // Just give an empty list, simplifies things.
         MinidumpModuleList::new()
     };
+    let unloaded_modules = if let Ok(module_list) = dump.get_stream::<MinidumpUnloadedModuleList>()
```

Comment:
I'm normally a fan of `if let` for this but given the formatting this might look nicer as a `match`.